### PR TITLE
Fix two unit tests of OperationChecker

### PR DESCRIPTION
**Pull request by Torch3333**
I found by chance small errors in two unit tests of the `OperationChecker`, this PR fixes them. Please see the inline comments for more details.

NB: the "docker" CI is failing because of a CVE

### CODE REVIEW

File: core/src/test/java/com/scalar/db/storage/common/checker/OperationCheckerTest.java [hunk 1]
```diff
@@ -537,7 +537,7 @@ public void whenCheckingScanOperationWithInvalidOrderings_shouldThrowIllegalArgu
     Key startClusteringKey = new Key(CKEY1, 2, CKEY2, "val1");
     Key endClusteringKey = new Key(CKEY1, 2, CKEY2, "val9");
     List<String> projections = Arrays.asList(COL1, COL2, COL3);
-    int limit = -10;
```
Comment (Torch3333) [inline core/src/test/java/com/scalar/db/storage/common/checker/OperationCheckerTest.java:540]: The unit test goal is to validate that a wrong ordering triggers an `IllegalArgumenException`
```
.withOrdering(new Scan.Ordering(CKEY1, Scan.Ordering.Order.DESC))
.withOrdering(new Scan.Ordering(CKEY2, Scan.Ordering.Order.DESC))
 ```
 Though, by mistake, the limit was set to a negative value, throwing an `IllegalArgumenException` before `operationChecker.check(scan)` reaches the step that validates the ordering. This made the test pass for the wrong reason. 
 With a valid positive limit, the test will now validate the wrong ordering.

File: core/src/test/java/com/scalar/db/storage/common/checker/OperationCheckerTest.java [hunk 2]
```diff
@@ -562,7 +562,7 @@ public void whenCheckingScanOperationWithInvalidOrderings_shouldThrowIllegalArgu
...
+    int limit = 10;
```
Comment (Torch3333) [inline core/src/test/java/com/scalar/db/storage/common/checker/OperationCheckerTest.java:565]: Ditto.

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Review by komamitsu — APPROVED** [review]
LGTM! Good catch!

**Review by feeblefakie — APPROVED** [review]
Good catch! Thank you!

**Pull request merged**